Desserts/Data/Mappers/RecipeMapper.swift

```diff
@@ -12,7 +12,7 @@ extension RecipeDTO {
             id: id,
             name: name,
             time: time,
-            level: level,
+            level: Level(rawValue: level)?.id ?? "",
```

Comment:
I'd like to suggest to define a constant to do this calculations and then just set the constant result here. To avoid to have this kind of calculations as part of the model creation

```swift
let level = Level(rawValue: level)?.id ?? ""

// then you just have to set the result 
level: level
```

that makes the code more readable

Reply:
Done!